.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,59 @@
+name: Deploy to Netlify
+
+on:
+  push:
+    branches:
+      - main
+      - init
+  repository_dispatch:
+    types: [zui-docs-update]
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy to Netlify
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build site
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: yarn
+      - name: Build
+        run: make build
+      # Popular action to deploy to Netlify:
+      # Docs: https://github.com/marketplace/actions/netlify-actions
+      - name: Deploy to Netlify
+        uses: nwtgck/actions-netlify@v1.2
+        with:
+          # Build output to publish to Netlify
+          publish-dir: ./build
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          production-branch: init
+          production-deploy: true
+        env:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+      - name: Check for broken links
+        uses: ruzickap/action-my-broken-link-checker@v2
+        with:
+          url: https://zui.brimdata.io
+          cmd_params:
+            --verbose
+            --buffer-size=8192
+            --max-response-body-size 1000000000
+            --header="User-Agent:Mozilla/5.0"
+            --header="Accept-Encoding:zstd,br,gzip,deflate"
+            --exclude algolia.net
+            --exclude www.googletagmanager.com
+            --exclude https://support.virustotal.com/hc/en-us/articles/115002146769-Comments
+            --exclude https://github.com/brimdata/brimcap#
+            --exclude https://github.com/brimdata/build-suricata.*#
+            --exclude https://github.com/brimdata/brimcap/wiki.*#
+
+#      - name: Trigger Algolia Crawler
+#        run: |
+#          curl -X POST -H "Content-Type: application/json" \
+#               --user "${{ secrets.ALGOLIA_CRAWLER_APP_ID }}:${{ secrets.ALGOLIA_CRAWLER_API_KEY }}" \
+#               "https://crawler.algolia.com/api/1/crawlers/${{ secrets.ALGOLIA_CRAWLER_ID }}/reindex"
```

Comment:
I'm intentionally not trying to do any Algolia crawling yet because I don't want to pollute the index that powers the Search at zed.brimdata.io with hits on the draft content at zui.brimdata.io. That said, here's a summary of my take on the Algolia search across the two sites.

When I shared with the team my findings on the limitations of the Docusaurus multi-instance support, we reached consensus that we should instead take the simple approach of having wholly separate docs sites. As I started down this path, it had the ripple effect of my creating a separate repo, and I briefly thought I'd also need to pursue a separate Algolia search config. However, @mattnibs helpfully nudged me toward researching if we could extend the existing config. Indeed, a definite benefit of having a single Search tool across both sites is that it removes any burden on the user having to initiate their searches in the "correct" site, e.g., if a new user is unfamiliar with how the Zed project provides the query language in Zui, they may start at zui.brimdata.io but their  searches for Zui language terms will offer them helpful pages in the Zed docs.

Based on my findings thus far, I think it should be possible to maintain one index. One very important step along this path was the discovery of the `externalUrlRegex` setting in the Algolia settings of the Docusaurus config (thanks @nwt!), since this allows the Zed doc hits found when initiating Search from zui.brimdata.io get hyperlinked properly (try it!) Therefore, when this PR merges and we start crawling both sites, we'll need to make that same `externalUrlRegex` change in zed-docs-site as well.

Beyond that, I see in our current Algolia crawler configs in their Editor where we would add elements to certain arrays to include the new site. If for some reason this doesn't work or isn't allowed due to some limitation of the free tier we're on, we could just pick that moment to start paying Algolia. We could certainly afford their charges of $1 per 1,000 requests/mo.

![image](https://user-images.githubusercontent.com/5934157/190462306-269d83d9-a5e7-4886-9237-8bf70777269c.png)